tests/graph/test_centrality.py

```diff
@@ -230,3 +230,46 @@ def test_custom_graph_unnormalized(self):
         expected = {0: 9, 1: 9, 2: 12, 3: 15, 4: 11, 5: 14, 6: 10, 7: 13, 8: 9, 9: 9}
         for k, v in centrality.items():
             self.assertAlmostEqual(v, expected[k])
+
+
+class TestDegreeCentrality(unittest.TestCase):
+    def setUp(self):
+        self.graph = rustworkx.PyGraph()
+        self.a = self.graph.add_node("A")
+        self.b = self.graph.add_node("B")
+        self.c = self.graph.add_node("C")
+        self.d = self.graph.add_node("D")
+        edge_list = [
+            (self.a, self.b, 1),
+            (self.b, self.c, 1),
+            (self.c, self.d, 1),
+        ]
+        self.graph.add_edges_from(edge_list)
+
+    def test_degree_centrality(self):
+        centrality = rustworkx.degree_centrality_py(self.graph)
```

Comment:
```suggestion
        centrality = rustworkx.degree_centrality(self.graph)
```
A reminder to call the non-type specific function once you implement it in `__init__.py`